helpers/clients.py

```diff
@@ -0,0 +1,74 @@
+from confluent_kafka import SerializingProducer, DeserializingConsumer
+from confluent_kafka.error import SerializationError
+from confluent_kafka.serialization import StringSerializer, StringDeserializer
+from confluent_kafka.schema_registry import SchemaRegistryClient
+from confluent_kafka.schema_registry.avro import AvroSerializer, AvroDeserializer
+
+from classes.user import User
+from classes.pageview import Pageview
+
+import yaml
+
+
+def config():
+	# fetches the configs from the available file
+	with open('./config/config.yaml', 'r') as config_file:
+		config = yaml.load(config_file, Loader=yaml.CLoader)
+
+		return config
+
+
+def sr_client():
+	# set up schema registry
+	sr_conf = config()['schema-registry']
+	sr_client = SchemaRegistryClient(sr_conf)
+
+	return sr_client
+
+
+def user_deserializer():
+	return AvroDeserializer(
+		schema_registry_client = sr_client(),
+		schema_str = User.get_schema(),
+		from_dict = User.dict_to_user
+		)
+
+
+def pageview_deserializer():
+	return AvroDeserializer(
+		schema_registry_client = sr_client(),
+		schema_str = Pageview.get_schema(),
+		from_dict = Pageview.dict_to_pageview
```

Comment:
```suggestion
		from_dict = Pageview.dict_to_mapping
```

otherwise, PageView consumer won't work

Reply:
Ah, changing the class code. It should be pageview.